css/main.css

```diff
@@ -51,6 +52,25 @@ header {
   color: white;
 }
 
+#mobile-nav {
+  display: none;
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  background: green;
+  color: #fff;
+  z-index: 10;
+  padding-top: 100px;
+}
+
+#mobile-nav a {
+  color: red;
+  border-bottom: solid 1px red;
+  padding-bottom: 20px;
+}
```

Comment:
- [ ]  Great job adding the styles to the mobile menu, But you are missing some requirements
     1. Menu options should be aligned to the left. and match the colors, fonts with the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
     2.  Your mobile menu design is totally different from the requirements, You are supposed to stick to the [figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Please follow the design on the template as much as possible.
     Your design
![Screenshot (35)](https://user-images.githubusercontent.com/83803414/152337324-fafce0c9-d917-403f-9912-8e59f8a14690.png)
      Figma design
![Screenshot (36)](https://user-images.githubusercontent.com/83803414/152337364-7e172d18-bb90-4024-a384-a716e3ef368f.png)